src/widgets/Number/components/Number.js

```diff
@@ -1,19 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import _round from 'lodash/round';
 import './Number.scss';
 
+const getRounded = decimal => _round(decimal, 2);
+const getChange = (current, previous) => (previous === 0 ? previous : (current - previous) / previous * 100); // eslint-disable-line no-mixed-operators
```

Comment:
Well, if previous is zero it's kind of hard to calculate change, as there is not base value. Everything multiplied by zero... you know :)